src/com/dotcms/rest/api/v1/user/UserResourceHelper.java

```diff
@@ -281,26 +282,23 @@ public void log(final String action, String message) {
 	 * @throws Exception if anything if wrong
      */
 	public List<Map<String, Object>> getLoginAsUser() throws Exception {
-		final Map<String, String> filterParams = map(
-				"start", "0",
-				"limit", "30",
-				"includeAnonymous", "false",
-				"includeDefault", "false");
 
-		List<Map<String, Object>> userList = (List) this.getUserList(null, "1", filterParams).get("data");
+		List<User> users = userAPI.getUsersByNameOrEmailOrUserID(StringPool.BLANK, 1, 30, false, false);
+
+		List<Map<String, Object>> userList = new ArrayList<>();
 		List<String> rolesId = new ArrayList<>();
```

Comment:
if you want (totally optional) in a similar way you do with map (from CollectionUtils) you can do list, such as

List<String> rolesId = list( roleAPI.loadRoleByKey(Role.ADMINISTRATOR).getId() , //Admin Roles
roleAPI.loadCMSAdminRole().getId()); //Login As Roles